app/controllers/concerns/form_attachment_create.rb

```diff
@@ -5,11 +5,28 @@ module FormAttachmentCreate
   include SentryLogging
 
   def create
+    debug_timestamp = Time.current.iso8601
+    if Flipper.enabled?(:hca_log_form_attachment_create)
+      log_message_to_sentry(
+        'begin form attachment creation',
+        :info,
```

Comment:
What will `info` be? Is there any chance of PII being in there?

Reply:
`:info` is the logger level, as opposed to `:warning` or `:error`. The only dynamic data we're logging will be below - a boolean for whether the `:file_data` param exists, the name of the file's class (if present), and the timestamp I generate above.
```  
        file_data_present: filtered_params[:file_data].present?,
        class_name: filtered_params[:file_data]&.class&.name,
        debug_timestamp:
```

Reply:
ohhh, got it. The log level 👍